client/js/commands/search.js

```diff
@@ -4,10 +4,15 @@ import store from "../store";
 import {router} from "../router";
 
 function input(args) {
+	if (!store.state.settings.searchEnabled) {
+		return true;
+	}
+
+	const {channel} = store.state.activeChannel;
```

Comment:
This seems like a fairly random stylistic change to include here and doesn't seem necessary.